azure-pipelines.yml

```diff
@@ -102,8 +102,47 @@ jobs:
                $(Slack.URL)
         condition: and(failed(), eq(variables['Build.SourceBranchName'], 'master'))
 
+  - job: Windows_signing
+    # Signing is a separate job so that we can make sure that we only sign on releases.
+    # Since the release check is run on Linux, we do not have access to that information
+    # in the regular Windows step.
+    dependsOn: [ "Windows", "Linux" ]
+    pool:
+      name: 'windows-pool'
+    condition: and(succeeded(), eq(dependencies.Linux.outputs['release.has_released'], 'true'))
+    variables:
+      unsigned-installer: $[ dependencies.Windows.outputs['publish.artifact-unsigned-windows-installer'] ]
+    steps:
+      - checkout: self
+        persistCredentials: true
+      - bash: |
+          set -euo pipefail
+          echo "##vso[task.setvariable variable=release_tag]$(cat VERSION)"
+        name: release
+      - task: DownloadPipelineArtifact@0
+        inputs:
+          artifactName: $(unsigned-installer)
+          targetPath: $(Build.StagingDirectory)/
+      - bash: |
+          set -euo pipefail
+          INSTALLER=daml-sdk-$(release_tag)-windows.exe
```

Comment:
You can use `$(cat VERSION)` here directly and save out a whole `bash` step.

Reply:
Good idea, changed!